ivi-layermanagement-examples/LayerManagerControl/CMakeLists.txt

```diff
@@ -43,7 +43,6 @@ SET(SRC_FILES
     src/analyze.cpp
     src/common.cpp
     src/control.cpp
-    src/demo.cpp
```

Comment:
We can add some similar testcase into our test framework. I understand that it is useful but LayerManagerControl is not the place for testing.